pkg/synchromanager/clustersynchro/resource_negotiator.go

```diff
@@ -117,13 +117,15 @@ func (negotiator *ResourceNegotiator) NegotiateSyncResources(syncResources []clu
 				}
 
 				storageConfig, err := negotiator.resourceStorageConfig.NewConfig(syncGVR)
+
 				if err != nil {
 					syncCondition.Reason = "SynchroCreateFailed"
 					syncCondition.Message = fmt.Sprintf("new resource storage config failed: %s", err)
 					groupResourceStatus.addSyncCondition(syncGVR, syncCondition)
 					continue
 				}
 
+				storageConfig.Namespaced = apiResource.Namespaced
```

Comment:
It might be better to pass Namespaced into `negotiator.resourceStorageConfig.NewConfig`, since namespaced is also required to generate storageconfig in kubeapiserver.

https://github.com/clusterpedia-io/clusterpedia/blob/d5cc72687a196f49c789b0aafe2b76c84e277bb6/pkg/kubeapiserver/restmanager.go#L293-L294

https://github.com/clusterpedia-io/clusterpedia/blob/d5cc72687a196f49c789b0aafe2b76c84e277bb6/pkg/kubeapiserver/restmanager.go#L266-L267

And namespaced is a required field in StorageConfig, as a parameter to `StorageConfigFactory.New*`, to prevent forgetting to set